### DISCUSSION TITLE
https 下的带宽性能

### DISCUSSION THREAD

**Discussion by 0zero-r** [alibaba/higress#1613] [Q&A]
- 内网带宽 1000M
- 内部有一个 download server，含有一个 2GB 大大文件。
- 使用 http 可以跑到 800M 左右
- 使用 https 只能跑到 200M 左右

---

这是符合预期的吗，如果不符合该如何优化？

**Comment by johnlanni** [comment]
符合预期的，https 的 tls 握手会极大影响性能，阿里云的 higress 托管版做了 tls 硬件加速，会有较大提升

**Reply by 0zero-r** [reply]
但是性能有接近 4 倍差距？我们用 nginx/kong 虽然也会降低性能，但没有这么夸张
